### PR TITLE
TG-1999 Config dependant browser caching

### PR DESCRIPTION
**Pull request by youwol-jdecharne**
- ✨ `youwol.utils.crypto.digest`
- ✨ `YouwolEnvironmentFactory` compute config digest
- ✨ `config_dependant_browser_caching`
- 📝 Pydoc for Features

TG-2125 #closed
TG-2126 #closed
TG-1999 #done

### CODE REVIEW

File: src/youwol/app/environment/models/models_features.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# standard library
+from dataclasses import dataclass
+
+
+@dataclass(frozen=True, kw_only=True)
```
Comment (youwol-guillaume) [inline src/youwol/app/environment/models/models_features.py:5]: We usually use `BaseModel` as base class to define configuration's related models, is using `dataclass` intended?

File: src/youwol/app/environment/models/models_features.py [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+class Features:
+    config_dependant_browser_caching: bool = False
```
Comment (youwol-guillaume) [inline src/youwol/app/environment/models/models_features.py:7]: In configuration's models we use camelCase for attribute's names.

Also, if the class need to be exposed in the API documentation, docstring is needed for the class and the attribute.

File: src/youwol/app/environment/models/models_config.py [hunk 3]
```diff
@@ -485,3 +486,5 @@ class Configuration(BaseModel):
     """
     Various handles for customization (*e.g.* middleware, commands, ...)
     """
+
```
Comment (youwol-guillaume) [inline src/youwol/app/environment/models/models_config.py:489]: To expose this attribute in the API documentation, docstring is needed.

File: src/youwol/app/environment/models/models_config.py [hunk 4]
```diff
@@ -485,3 +486,5 @@ class Configuration(BaseModel):
...
+    features: Features = Features()
```
Comment (youwol-jdecharne) [inline src/youwol/app/environment/models/models_config.py:490]: Add docstring here

**Review by youwol-guillaume — APPROVED** [review]

**Pull request merged**